src/core/Expander/ExpanderGroup/ExpanderGroup.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { Component, ReactNode } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { withSuomifiDefaultProps } from '../../theme/utils';
+import { TokensProp, InternalTokensProp } from '../../theme';
+import { HtmlDiv, HtmlButton, HtmlButtonProps, HtmlSpan } from '../../../reset';
+import { baseStyles } from './ExpanderGroup.baseStyles';
+import { VisuallyHidden } from '../../../components';
+
+const baseClassName = 'fi-expander-group';
+const openClassName = `${baseClassName}--open`;
+const expandersContainerClassName = `${baseClassName}_expanders`;
+const openAllButtonClassName = `${baseClassName}_all-button`;
+interface InternalExpanderGroupProps {
+  /** Expanders and optionally, other ReactNodes */
+  children: ReactNode;
+  /** 'Open all' button text */
+  OpenAllText: string;
+  /** 'Close all' button text */
+  CloseAllText: string;
+  /** 'Open all' button text for screen readers, hides OpenAllText for screen readers if provided */
+  AriaOpenAllText?: string;
+  /** 'Close all' button text for screen readers, hides CloseAllText for screen readers if provided */
+  AriaCloseAllText?: string;
+  /** Custom classname to extend or customize */
+  className?: string;
+  /** Open/Close all button props */
+  toggleAllButtonProps?: Omit<
+    HtmlButtonProps,
+    | 'onClick'
+    | 'onMouseDown'
+    | 'onMouseUp'
+    | 'onKeyPress'
+    | 'onKeyUp'
+    | 'onKeyDown'
+  >;
+}
+
+interface ExpanderOpenStates {
+  [key: string]: boolean;
+}
+
+type ExpanderGroupTargetOpenState = {
+  targetOpenState: boolean;
+};
+
+interface ExpanderGroupState {
+  /** Expanders that are open */
+  expanders: ExpanderOpenStates;
```

Comment:
Just a small thing, but wasn't it so that this will always hold the state of the expanders not matter closed or opened?
If so, then the comment might be little misguiding.

Reply:
Changed to `Expanders by id's with current open state`